File: in_Java/1266 Minimum Time Visiting All Points.java
class Solution {
    public int minTimeToVisitAllPoints(int[][] points) {
        int res = 0;
        int a = points[0][0];
        int b = points[0][1];
        for(int i = 1; i < points.length; i++){
            int x = points[i][0];
            int y = points[i][1];
            res += Math.max(Math.abs(x-a), Math.abs(y-b));
            a = x;
            b = y;
        }
        return res;
    }
}

File: in_Python/1266 Minimum Time Visiting All Points.py
class Solution(object):
    def minTimeToVisitAllPoints(self, points):
        """
        :type points: List[List[int]]
        :rtype: int
        """
        res = 0
        a = points[0][0]
        b = points[0][1]
        for i in range(1,len(points)):
            aa = points[i][0]
            bb = points[i][1]
            res += max(abs(aa-a),abs(bb-b))
            a = aa
            b = bb
        return res